grails-app/views/admin/adminReports.gsp

```diff
@@ -27,17 +27,39 @@
 <a id="orgDataDownload" class="btn btn-sm" href="#">Download Organisation Report Data</a>
 
 <h4>Management unit report download:</h4>
-<div class="form-group">
-    <label class="control-label">Select reporting period: </label>
-    <div class="controls">
-        <select id="reportPeriodOfManagementUnit" class="form-control form-control-sm input-medium">
-            <g:each var="financialYear" in="${reportsPeriodsOfManagementUnit}">
-                <option value="startDate=${financialYear}-07-01&endDate=${financialYear+1}-06-30">01 July ${financialYear} - 30 June ${financialYear+1} </option>
-            </g:each>
-        </select>
+<form id="mu-report-selector">
```

Comment:
This dropdown is still on the page, however it doesn't appear to do anything?  For example, it doesn't modify the date selection if you pick a date range from the dropdown.  I suggest we delete this.